src/hooks/useFn.js

```diff
@@ -0,0 +1,9 @@
+import { useState } from 'react';
+
+export default function useFn(fn) {
```

Comment:
hmm i'm not sure we need a hook specifically for this. if we're storing a function in state, usually it's fine to keep key it in an object like `setState({ fnc })`

Reply:
Yes that works! Will change